config/main.py

```diff
@@ -973,6 +1016,14 @@ def portchannel_member(ctx):
 def add_portchannel_member(ctx, portchannel_name, port_name):
     """Add member to port channel"""
     db = ctx.obj['db']
+    if sonic_device_util.is_multi_npu():
```

Comment:
this is logic is not straightforward.

why not just connect to the db in ns, and check if port_name is in the port_table, it will work for both single asic and multi-asic.

Reply:
The logic here is to make sure that the member interface which is getting added into the port channel is in fact present in the same namespace the user inputted with the -n namespace argument.
I have introduced this check only for the multi-asic platforms where -n namespace parameter is mandatory with these port-channel commands.

Reply:
I updated to use the existing function interface_name_is_valid(), as in this case we already have the details of which config db to connect to.